JUST.net/ComparisonHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace JUST
+{
+    internal static class ComparisonHelper
+    {
+        public static bool Equals(object x, object y, JUSTContext context)
```

Comment:
If you only need EvaluationMode, there's no point in passing JUSTContext here, pass only EvaluationMode